tests/foreman/smoke/test_ui_smoke.py

```diff
@@ -46,7 +46,7 @@ class SmokeTestCase(UITestCase):
     """End-to-end tests using the ``WebUI``."""
 
     def test_positive_find_default_org(self):
-        """@Test: Check if :data:`robottelo.constants.DEFAULT_ORG` is present
```

Comment:
Removed `:` intentionally because:
* testimony parsing is affected because of that
* also to make meaningful test names in robottelo

This comment applies to all similar changes in this PR.